src/jyoti/Assignment23.java

```diff
@@ -0,0 +1,45 @@
+/*
+Assignment - 23 : 14th Sep'2023
+Write a program to find count of Uppercase, lowercase, digits and special characters.
+String str = "TecHNo_cR6ediTs";
+output : Uppercase -> 5
+         Lowercase -> 8
+		 Digit -> 1
+		 Special char count -> 1
+*/
+package jyoti;
+public class Assignment23{
+
+
+	void stringCheck(String input){
+		int uCount=0;
+		int lCount=0;
+		int dCount=0;
+		int spaceCount=0;
+		int sCount=0;
+
+		for (int index=0;index<input.length();index++){
+			if (Character.isUpperCase(input.charAt(index))){
```

Comment:
It would be better if you create char ch = input.charAt(index) outside of condition, so you no need to write it again and again, you have to just write ch